src/Courses/CourseDetailsTabs/StatisticsTab.tsx

```diff
@@ -24,18 +31,34 @@ enum StatisticsView {
     ATTEMPTS = 'ATTEMPTS',
 }
 
+enum StatisticsViewFilter {
+    UNITS_FILTERED = 'UNITS_FILTERED',
```

Comment:
You should use `UNITS_${FILTERED_STRING}` so that it remains consistent, but it's not a big deal.

Reply:
Yeah I added the string for _FILTERED after the fact
The problem is I want the value to match the key, not sure if  enums will let me use variables in the key

Reply:
I get this error if I try to put it in the key: `Computed property names are not allowed in enums.ts`
So the question is do we want the key and value to be a strict copy and paste or do we want the value to use the constant 🤔